src/osm_parser.rs

```diff
@@ -227,26 +222,20 @@ pub fn parse_osm_data(
             for &node_id in node_ids {
                 if let Some(node) = nodes_map.get(&node_id) {
                     nodes.push(node.clone());
+                } else {
+                    println!("bro what");
                 }
             }
         }
 
-        if !nodes.is_empty() {
```

Comment:
These various changes is so we don't accidentally exclude a node/way that's being used in our relation